es6-modules/src/lib/index.js

```diff
@@ -0,0 +1,9 @@
+import noop from './noop';
+import toArray from './to-array';
+import createElement from './create-element';
+
+export default {
```

Comment:
Here you want to get rid of the `default` keyword. You want to create named exports.